src/app/component/MyAccountPasswordForm/MyAccountPasswordForm.style.scss

```diff
@@ -0,0 +1,14 @@
+.MyAccountPasswordForm{
+    &-Wrapper{
+        display: flex;
+        justify-content: center;
+        @include after-mobile{
+            padding-top: .5rem; // var(--popup-content-padding) - 1rem
+        }
+        button{
+            @include mobile{
+                width:100%;
+            }
+        }
+    }
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/app/ScandiPWA/base-theme/pullRequest?prid=4698981)